src/Throttlers/CacheThrottler.php

```diff
@@ -177,9 +178,17 @@ protected function hitRedis()
                'if v>1 then return v '.
                'else redis.call(\'setex\', KEYS[1], ARGV[1], 1) return 1 end';
 
-        $this->number = $this->store->connection()->eval(
-            $lua, 1, $this->store->getPrefix().$this->key, max(1, $this->time * 60));
+        $this->number = $this->store->connection()->eval($lua, 1, $this->computeRedisKey(), $this->time);
```

Comment:
Removing `* 60` part is a bug, I think. The lau script usage setex which expects arg in seconds. $this->time is in minutes.

Will need redis driver specific tests to avoid such, as no longer using abstracted interface for underlying driver. Maybe minor refactor to avoid if..else (driver specific) logic.

Reply:
> Thanks for the PR. I'm just reviewing it now. Why do we need the max function on the time?

Just being defensive, in case someone puts negative, float values in config. RedisStore implmentation put() also does the same `(int) max(1, $minutes * 60)`.



> It's a bit limiting to say you can't have less than 1 minute?

Didn't understand this?

Reply:
Filed https://github.com/GrahamCampbell/Laravel-Throttle/pull/78